07LessonSeven/in-class-project/index.html

```diff
@@ -3,8 +3,58 @@
 
 <head>
     <title>Template</title>
+    <link type="text/css" rel="stylesheet" href="css/my-style.css">
+
 </head>
 
-<body></body>
+<body>
+
+      <div class="image">
+         <img class="image" src="img/nasa-background.jpg" alt="" />
+         <p class="text"><strong>NASA</strong></p>
+      </div>
+
+      <section class="container">
+
+         <div class="box1">

```

Comment:
your container is collapsed! go back to the textbook to get the "clearfix" class and add it to the container to prevent it.